docs/stability-tiers.md

```diff
@@ -259,32 +410,43 @@ well-maintained, tested well, but don't necessarily meet the stringent criteria
 for Tier 1. Features in this category may already be "production ready" and safe
 to use.
 
-Tier 2 features include:
+Tier 2 features, organized by category, include:
+
+* **Target**: tests are run in CI for this platform with a compiler, either
+  directly or via emulation. This ensures that all changes to Wasmtime are
+  tested against this target, and this additionally means that all applicable
+  Tier 1/2 features are tested on this target in CI.
+
+* **Target**: Complete implementations for anything that's part of Tier 1. For
+  example all Tier 2 targets must implement all of the applicable Tier 1
+  WebAssembly proposals, and all Tier 2 features must be implemented on all Tier
+  1 targets.
```

Comment:
Small note on formatting: could these two be included under a single Target bullet point? e.g., 

* Target
  * Tests are run ... 
  * Complete implementation ...


In the rendered version, it took me a bit to understand the current formatting, but if I'm understanding correctly, they are meant to be two separate points under the same Target classification (similar above with Maintenance).